package.json

```diff
@@ -47,5 +47,9 @@
   },
   "jest": {
     "preset": "react-native"
-  }
+  },
+  "files": [
+    "ios",
+    "react-native-home-indicator.podspec"
+  ]
```

Comment:
i think we should also include the main `index.js` to this list

Reply:
'main' is included by default: [link](https://docs.npmjs.com/files/package.json#files)

I've tested my fork with the changes in a fresh 0.60.4 repo, and the npm install + pod install work as expected.

Reply:
ok, good to know